staging/src/k8s.io/apiserver/pkg/util/flowcontrol/apf_controller.go

```diff
@@ -68,6 +68,14 @@ const timeFmt = "2006-01-02T15:04:05.999"
 // undesired becomes completely unused, all the config objects are
 // read and processed as a whole.
 
+// The funcs in this package follow the naming convention that the suffix
+// "Locked" means the relevant mutex must be locked at the start of each
+// call and will be locked upon return.  For a configController, the
+// suffix "ReadLocked" stipulates a read lock while just "Locked"
+// stipulates a full lock.  Absence of either suffix means that either
+// (a) the lock must NOT be held at call time and will not be held
```

Comment:
I look at it this way: using "Locked" gives some information.  Not using "Locked" is the complement of that.  Yes, it might be nice to distinguish cases in that complement.  But do you really want to see more stuff added to more func names?

Reply:
Also, this runs into trouble when implementing an interface.  You do not want an implementation distinction (between "doesn't care about the implementation's lock" and "must not hold the implementation's lock") to appear in the func names in the interface.

Reply:
I think it would be better to describe the situation like this: "For functions without either suffix, look at the function comment for any locking requirements."

Reply:
I think that's the obvious conclusion from the existing text, but would be fine with adding that explicit statement.

Reply:
How would you recommend proceeding, from the current state of the codebase and PRs?

Reply:
I don't think it's worth changing this since it already merged, I was just (attempting to) note that the comment isn't very helpful, but is written in language that makes it seem like it should be helpful.